tools/scripts/clean-build.js

```diff
@@ -0,0 +1,64 @@
+#!/bin/env node
+let path = require('path'), fs=require('fs');
+
+function fromDir(startPath, filter, callback){
+
+    if (!fs.existsSync(startPath)){
+        console.log("no dir ",startPath);
+        return;
+    }
+
+    let files = fs.readdirSync(startPath);
+    for (let i = 0; i < files.length; i++) {
+        let filename = path.join(startPath,files[i]);
+        let stat = fs.lstatSync(filename);
+        if (stat.isDirectory()) {
+            fromDir(filename, filter, callback);
+        }
+        else if (filter.test(filename)) {
+            callback(filename, files[i])
+        }
+    }
+}
+
+function justPrint(filePath, fileName){
+    console.log('-- found: ', fileName.split('.')[0]);
```

Comment:
do we need this console log?

Reply:
It provides some good detail as to which assets are being removed

Reply:
In the event that something fails in the pipeline and it is this we have a clear indicator

Reply:
Just need to make sure this doesn't pop up in the browser console for production